codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/ServerServiceGeneratorV2.kt

```diff
@@ -213,6 +224,17 @@ class ServerServiceGeneratorV2(
                     }
                 }
             }
+
+            impl<$builderGenerics, NewPlugin> #{SmithyHttpServer}::plugin::Pluggable<NewPlugin> for $builderName<$builderGenerics> {
```

Comment:
Having a `,` here in `$builderGenerics, NewPlugin` might cause problems when `$builderGenerics` is empty? I'm not entirely sure. It might be worth generating this using `.joinToString()` so we don't get a comma in when `$builderGenerics` is empty.

Reply:
Same below with `$builderGenericsNoPlugin, aws_smithy_http_server::plugin::PluginStack<$pluginName, NewPlugin>`.

I'm not sure this is worth it though - there might be a ton of other instances where this breaks in the case of no operations.

Reply:
It breaks the operation registry too

Reply:
Related: https://github.com/awslabs/smithy-rs/pull/1369